src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -177,6 +142,46 @@ private void launchNode(SchedulerDriver driver, Offer offer,
     driver.launchTasks(Arrays.asList(offer.getId()), tasks);
   }
     
+  private ExecutorInfo createExecutor(String taskIdName, String nodeName, String executorName) {
+    int confServerPort = conf.getConfigServerPort();
+    List<Resource> resources = getExecutorResources();
+    ExecutorInfo executorInfo = ExecutorInfo.newBuilder()
+        .setName(nodeName + " executor")
+        .setExecutorId(ExecutorID.newBuilder().setValue("executor." + taskIdName).build())
+        .addAllResources(resources)
+        .setCommand(CommandInfo.newBuilder()
+        .addAllUris(Arrays.asList(
+            CommandInfo.URI.newBuilder().setValue(
+                conf.getExecUri())
+                .build(),
+            CommandInfo.URI.newBuilder().setValue(
+                String.format("http://%s:%d/hdfs-site.xml", localhost, confServerPort))
+                .build()))
+            .setEnvironment(Environment.newBuilder()
+                .addAllVariables(Arrays.asList(
+                Environment.Variable.newBuilder()
+                    .setName("HADOOP_OPTS")
+                    .setValue(conf.getJvmOpts()).build(),
+                Environment.Variable.newBuilder()
+                    .setName("HADOOP_HEAPSIZE")
+                    .setValue(String.format("%d", conf.getHadoopHeapSize())).build(),
+                Environment.Variable.newBuilder()
+                    .setName("HADOOP_NAMENODE_OPTS")
+                    .setValue("-Xmx" + conf.getNameNodeHeapSize() + "m").build(),
+                Environment.Variable.newBuilder()
+                    .setName("HADOOP_DATANODE_OPTS")
+                    .setValue("-Xmx" + conf.getDataNodeHeapSize() + "m").build(),
+                Environment.Variable.newBuilder()
+                    .setName("EXECUTOR_OPTS")
+                    .setValue("-Xmx" + conf.getExecutorHeap() + "m").build())))
+                    .setValue(
+                        "env ; cd hdfs-mesos-* && exec java $HADOOP_OPTS $EXECUTOR_OPTS " +
```

Comment:
When I'm outputting env information to facilitate debugging I like to pipe it through sort so that it's easier to skim read. `evn | sort`